doc/api/crypto.md

```diff
@@ -2800,6 +2800,46 @@ threadpool request. To minimize threadpool task length variation, partition
 large `randomFill` requests when doing so as part of fulfilling a client
 request.
 
+### `crypto.randomInt([min, ]max[, callback])`
+<!-- YAML
+added:
+  - v14.8.0
```

Comment:
```suggestion
  - REPLACEME
```

(This is a “magical” keyword that is replaced automatically during the release process)

Reply:
Oops, haha!